frontend/src/component/operator-wizard/OperatorWizard.js

```diff
@@ -350,7 +355,42 @@ export default class OperatorWizard extends Component {
   getJsonSchemaPathForYaml(jsonPath) {
     return jsonPath.slice(2, jsonPath.length);
   }
+  addExternalDBEnvVars(jsonObject) {
+    const tempEnv = [
+      { name: EXTENSIONS_IMAGE_KEY, value: EXTENSIONS_IMAGE_VALUE },
+      {
+        name: EXTENSIONS_IMAGE_NAMESPACE_KEY,
+        value: EXTENSIONS_IMAGE_NAMESPACE_VALUE
+      }
+    ];
+
+    var server;
+    var obj = Dot.object(jsonObject);
+    if (
+      obj.spec !== undefined &&
+      obj.spec.objects !== undefined &&
+      obj.spec.objects.servers !== undefined
+    ) {
+      let servers = obj.spec.objects.servers;
+
+      for (server in servers) {
```

Comment:
We can say "let server in servers" and remove the "var server", and we may use map() here.

Reply:
updated